.jenkins

```diff
@@ -82,6 +80,10 @@ pipeline {
                             sh 'ccache --show-stats'
                             xunit([CTest(deleteOutputFiles: true, failIfNotNew: true, pattern: 'build/Testing/**/Test.xml', skipNoTestFiles: false, stopProcessingIfError: true)])
                         }
+                        success {
+                            sh 'cd build && make install'
+                            sh 'rm -rf test_install && mkdir -p test_install && cd test_install && cmake -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=clang++ -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$ARBORX_DIR" ../examples && make VERBOSE=1 && make test'
```

Comment:
`make -j8`? and what is `make test`?

Reply:
> `make -j8`?

There is only one cpp file at the moment.

> what is `make test`?

CMake create this rule by default and it calls CTest.